core-services/prow/02_config/_config.yaml

```diff
@@ -469,6 +469,7 @@ tide:
     - openshift/verification-tests
     - openshift/vertical-pod-autoscaler-operator
     - openshift/windows-machine-config-operator
+    - openshift-knative/serverless-operator
```

Comment:
This only adds `tide` for release branches -- you want another one up as well

Reply:
Really? This section has `excludedBranches` so I supposed all other branches would be included. Anyway, we currently only have the `master` branch. What should I do to enable the tide plugin for the master branch? Thanks

Reply:
Err so it only adds it for non-release, my bad. Do you not expect to have release branches?

Reply:
We expect to have release branches later. But they will be probably named differently from what we have in `excludedBranches`. So, I believe we should be fine. Please correct me if I'm missing something here.